### PR TITLE
Bump gopkg.in/yaml.v3

### PR DESCRIPTION
**Pull request by AlekSi**
See https://github.com/advisories/GHSA-hp87-p4gw-j4gq.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ linters-settings:
       - FIXME
       - HACK
   goheader:
+    # Please do not update it, including bumping a year.
```
[the remark this answers is not in the document]
Comment (AlekSi) [reply .golangci.yml:35]: No, we should not give legal advice. That's just a reminder for us, without rationale.

[the remark this answers is not in the document]
Comment (seeforschauer) [reply .golangci.yml:35]: > No, we should not give legal advice. That's just a reminder for us, without rationale.


I don't understand this part of the code in the project then.
Then it's something that I have no idea how to maintain.

[the remark this answers is not in the document]
Comment (seeforschauer) [reply .golangci.yml:35]: Unfortunately, it looks like an instruction without any explanation. 
It infers meaningless behavior and also reducing of the initiatives.

**Review by seeforschauer — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**